### PR TITLE
Adding ZCOMBO-G, take 2

### PR DESCRIPTION
**Pull request by richardtallent**
This replaces https://github.com/OpenZWave/open-zwave/pull/2419, I ended up in git hell and just ended up reverting and making the changes again.

I was able to get `make` to run by moving the repo to a path that didn't include a space in it. Everything seems to validate.

I'm not entirely sure about the Notification Class configuration. The list of notifications in the spec for this product (https://www.firstalert.com/themencode-pdf-viewer/?file=https://www.firstalert.com/wp-content/uploads/2020/06/BRK_ZCOMBO_Z-Wave_Specification_11.0.pdf) doesn't align with any list in `Localization.xml` under `COMMAND_CLASS_NOTIFICATION`. I don't know if that's expected or not, didn't find any documentation on updating this file for new devices.

### CODE REVIEW

**Comment by richardtallent** [comment]
Just a follow-up, I've been using this about a week now with three of these devices, and it seems to be working fine. Hope this can be rolled into the next official update...

File: config/firstalert/zcombo-g.xml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    <MetaDataItem name="ExclusionDescription">1. Slide battery door open.
+2. Insert batteries checking the correct orientation.
+3. Press and hold the test button. Keep it held down as you slide the battery drawer closed. You may then release the button.</MetaDataItem>
+    <MetaDataItem name="ResetDescription">If the device is powered up with the test button held down for 10+ seconds, the device will reset all Z-Wave settings and leave the network.
+Upon completion of the Reset operation, the LED will glow and the horn will sound for ~1 second.
+Please use this procedure only when the network primary controller is missing or otherwise inoperable.
+    </MetaDataItem>
+    <MetaDataItem id="0003" name="ZWProductPage" type="0001">https://products.z-wavealliance.org/products/3886/</MetaDataItem>
+    <MetaDataItem id="0003" name="FrequencyName" type="0001">U.S. / Canada / Mexico</MetaDataItem>
+    <MetaDataItem id="0003" name="Identifier" type="0001">ZCOMBO-G</MetaDataItem>
```
Comment (Socalix) [inline config/firstalert/zcombo-g.xml:23]: The manufacturer's data here https://products.z-wavealliance.org/products/3886/xml
for this device is showing the following values:
```
  <Identifier>ZCOMBO</Identifier>
  <Name>ZCOMBO-G</Name>
```

I suggest that in order to align with these values, this PR will have:
```
<MetaDataItem name="Name">ZCOMBO-G Battery Smoke/CO Alarm</MetaDataItem>
<MetaDataItem id="0003" name="Identifier" type="0001">ZCOMBO</MetaDataItem>
```

**Comment by richardtallent** [comment]
There's already a First Alert device with a `ZCOMBO` Identifier. Will a duplicate Identifier value create issues? If not, sounds good to me!

**Comment by kpine** [comment]
It won't cause any problems. It's just metadata that applications can use if they want.

**Comment by Fishwaldo** [comment]
Thanks!

**Pull request merged**